common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
I think that would be something new because i'm not able to see that in existing code.